src/harness/sas_interface.py

```diff
@@ -472,6 +472,16 @@ def TriggerFullActivityDump(self):
     """
     pass
 
+  @abc.abstractmethod
+  def InjectPeerSas(self, request):
+	"""SAS admin interface to Inject Peer Sas
```

Comment:
Please fix indentation here and on line 483 to match style with other functions.